### ISSUE TITLE
[Examples] Add example for Progressive Web Apps

### ISSUE THREAD

**Issue by lfades** [vercel/next.js#15356]
We currently don't mention PWAs in any part of our docs, and PWAs have been gaining popularity. An example called `progressive-web-app` could be quite useful.

I'm not sure about how PWAs are currently build with Next.js, but [`next-pwa`](https://github.com/shadowwalker/next-pwa) seems to be doing a good job. The example should feature static pages, without a custom server.

Once the example is done, add it to our FAQ page!

**Comment by eKhattak** [comment]
I can take this one

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
